espnet2/bin/enh_inference_sidon.py

```
@@ -0,0 +1,186 @@
+    parser.add_argument("--wav_scp", required=True)
+    parser.add_argument("--output_dir", required=True)
+    parser.add_argument("--device", default="cuda")
+    parser.add_argument("--chunk_sec", type=float, default=20.0)
+    parser.add_argument("--overlap_sec", type=float, default=0.5)
+    return parser
+
+
+def _read_audio(value: str, sample_rate: int = 16000):
+    value = value.strip()
+    if value.endswith("|"):
+        process = subprocess.run(
```

Ruff (0.16.3)

[error] `subprocess` call with `shell=True` identified, security issue

(S602)